nservicebus/pipeline/aborting.md

```diff
@@ -7,6 +7,8 @@ tags:
 
 ## From inside a Handler
 
+NOTE: In Version 6, the IBus interface has been deprecated and removed. Use the `DoNotContinueDispatchingCurrentMessageToHandlers` method on the `IMessageHandlerContext` interface instead. For more information, please consult the [upgrade guide](/nservicebus/upgrades/5to6.md).
```

Comment:
This should be the definitive source for aborting. The upgrade guide should reference this page not the other way around.